design-proposals/mass_machine_type_transition.md

```diff
@@ -0,0 +1,265 @@
+﻿# Overview
```

Comment:
one general question on this: We already have a process which is taking care of reporting and updating virt-launcher images when they are outdated. Would this just be another attribute to trigger these already established paths? Here the PR I mean: https://github.com/kubevirt/kubevirt/pull/4686

Reply:
In this case, live migration is not going to be able to accomplish the change, since it's outright changing the machine type of the VM. A restart of qemu is unavoidable.